src/components/AskAQuestion/index.tsx

```diff
@@ -7,14 +7,25 @@ import DiscussionModalWrapper from 'views/ModelPlan/Discussions/DiscussionModalW
 
 type AskAQuestionType = {
   modelID: string;
-  opNeeds?: boolean;
+  renderTextFor?: 'need' | 'solution' | 'status';
 };
 
-const AskAQuestion = ({ modelID, opNeeds }: AskAQuestionType) => {
+const AskAQuestion = ({ modelID, renderTextFor }: AskAQuestionType) => {
   const { t } = useTranslation('discussions');
   const { t: o } = useTranslation('itSolutions');
   const [isDiscussionOpen, setIsDiscussionOpen] = useState(false);
 
+  const renderText = (text: string | undefined) => {
+    switch (text) {
+      case 'need':
+        return o('notSureWhatToDoNext');
+      case 'status':
+        return o('helpTiming');
+      default:
+        return o('helpChoosing');
+    }
+  };
+
```

Comment:
As as readability point, this should include a case for 'solution'.  The default can either be an empty string something like:

```
case 'solution':
      default:
        return o('helpChoosing');
```